docker-bake.hcl

```diff
@@ -10,6 +10,10 @@ variable "DOCS_SITE_DIR" {
   default = "public"
 }
 
+variable "DRY_RUN" {
+  default = false
```

Comment:
Perhaps `default = nil` (or is it `null`?) and set the default in the Dockerfile (`ARG DRY_RUN=false`)? Or does it not need a default at all (i.e., if not set, then the default for a bool would be `false`?)

Reply:
yeah setting the default in the Dockerfile is more idiomatic 👍🏻 will change it